src/test/java/lv/ctco/javaschool/app/boundary/TripApiTest.java

```diff
@@ -79,4 +85,31 @@ void setTripPlaces() {
         tripApi.setTripPlaces(jsonObject, "1");
         verify(tripStore, times(1)).setTripPlaces(2, "1");
     }
+
+    @Test
+    @DisplayName("Check getting sorted list of UserLoginDto and calling tripStore.findTripsById() and userStore.findUsersByTrip()")
+    void getTripPassengers() {
+        List<UserLoginDto> userLoginDtos = new ArrayList<>();
+        UserLoginDto userLoginDto1 = new UserLoginDto("bastard", "pass1", "Hans", "Landa", "1111111");
+        UserLoginDto userLoginDto2 = new UserLoginDto("vader", "pass2", "Anakin", "Skywalker", "2222222");
+        UserLoginDto userLoginDto3 = new UserLoginDto("brother", "pass3", "Danila", "Bagrov", "3333333");
+        Collections.addAll(userLoginDtos, userLoginDto2, userLoginDto3, userLoginDto1);
+        List<User> users = new ArrayList<>();
+        User user1 = new User("bastard", "pass1", "Hans", "Landa", "1111111");
+        User user2 = new User("vader", "pass2", "Anakin", "Skywalker", "2222222");
+        User user3 = new User("brother", "pass3", "Danila", "Bagrov", "3333333");
+        Collections.addAll(users, user1, user2, user3);
+        Trip trip = new Trip();
+        when(tripStore.findTripById(1L)).thenReturn(Optional.of(trip));
+        when(userStore.findUsersByTrip(trip)).thenReturn(users);
+        tripApi.getTripPassengersByTripId("1");
```

Comment:
What about negative testing? Could you please add tests for 
* ` tripApi.getTripPassengersByTripId("one");`
* `tripApi.getTripPassengersByTripId("42");` with `when(tripStore.findTripById(1L)).thenReturn(Optional.empty());`

Reply:
I added the required tests.

Reply:
Yesterday I mean adequate exception handling instead of redundant tests:
* You could change `@PathParam("id") String tripId` to `@PathParam("id") Long tripId` to avoid manually cast String to Long. In case of string value in path Jersey will trigger 404 response. Less code from your side.
* Please use exception mapper (or directly Response object) for NoSuchElementException. You need to return 404 instead of 500 in such cases.

Reply:
- changed @PathParam("id") String tripId to @PathParam("id") Long tripId
- added custom UserNotFoundException and the corresponding UserNotFoundExceptionMapper
- rewrote redundant tests